src/components/landingPage/LandingPage.scss

```
@@ -0,0 +1,33 @@
+@import "../../constants.scss";
+
+.headline-image
+{
+    width: 100%;
+}
+
+.landing-page-button-menu
+{
+    height:100px; //feel like this should be changed to be more responsive - is it a site wide question for everyone?
```

What do you mean by this comment?

Also, prefer spaces after colons:
```suggestion
    height: 100px; //feel like this should be changed to be more responsive - is it a site wide question for everyone?
```

---

Can't remember now, i have taken it out